### PR TITLE
Add CHANGELOG for v0.42.0

### PR DESCRIPTION
**Pull request by misraved**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+
+- New tables added
+  - [aws_ec2_managed_prefix_list](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_ec2_managed_prefix_list) ([#813](https://github.com/turbot/steampipe-plugin-aws/pull/813))
+  - [aws_vpc_peering_connection](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_vpc_peering_connection) ([#814](https://github.com/turbot/steampipe-plugin-aws/pull/814))
+
+_Enhancements_
+
+- Added column `prefix_list_id` to `aws_vpc_security_group_rule` table ([#801](https://github.com/turbot/steampipe-plugin-aws/pull/801))
+- Added column `compliance_by_config_rule` to `aws_config_rule` table ([#817](https://github.com/turbot/steampipe-plugin-aws/pull/817))
+- Added additional optional key quals, filter support, and context cancellation handling to `Access Analyzer`, `ACM`, `API Gateway`, `Application Auto Scaling`, `Audit manager`, `Backup`, `Cloud Control`, `CloudFormation`, `CloudFront`, `CloudWatch`, `CodeBuild`, `CodeCommit`, `CodePipeline`, `Config`, `DAX`, `Directory Service`, `DMS`, `DynamoDB` and `EBS` tables   ([754](https://github.com/turbot/steampipe-plugin-aws/pull/754))
```
Comment (cbruno10) [inline CHANGELOG.md:13]:
```suggestion
- Added additional optional key quals, filter support, and context cancellation handling to `Access Analyzer`, `ACM`, `API Gateway`, `Application Auto Scaling`, `Audit manager`, `Backup`, `Cloud Control`, `CloudFormation`, `CloudFront`, `CloudWatch`, `CodeBuild`, `CodeCommit`, `CodePipeline`, `Config`, `DAX`, `Directory Service`, `DMS`, `DynamoDB` and `EBS` tables ([754](https://github.com/turbot/steampipe-plugin-aws/pull/754))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+- Added an example query in the `aws_rds_db_instance` table document ([#806](https://github.com/turbot/steampipe-plugin-aws/pull/806))
```
Comment (cbruno10) [inline CHANGELOG.md:14]:
```suggestion
- Added an example query for listing MySQL instances with SSL disabled in the `aws_rds_db_instance` table document ([#806](https://github.com/turbot/steampipe-plugin-aws/pull/806))
```

Comment (cbruno10) [reply CHANGELOG.md:14]: Also, in the table doc, the desc. should read `### List MySQL instances with SSL disabled in assigned parameter group` (https://github.com/turbot/steampipe-plugin-aws/pull/806/files#diff-3d9e194008ec43e1eb616a90e1dae15d5b2f7f36df9a760e347bf8dab751ae9eR108), can you please update that before releasing?

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comments, thanks!

**Review by cbruno10 — APPROVED** [review]

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**